docs/pages/enterprise/getting-started.mdx

```diff
@@ -228,12 +228,28 @@ role only allows SSH logins as `root@host`.
 ```yaml
 kind: role
 metadata:
-  name: access
+  name: access-sample
 spec:
   allow:
-    logins:
-    - '{{internal.logins}}'
-    - root
+    # The logins array defines the OS/UNIX logins a user is allowed to use.
+    # Using the variable internal.logins allows including user logins
```

Comment:
I would reframe this explanation around what substitutions Teleport performs on the `internal.logins` template variable. If we only say what "Using the variable internal.logins allows", it's not clear how it works or how someone would use it.